docs/api/vast-client.md

```diff
@@ -117,6 +117,8 @@ By default the fully parsed `VASTResponse` contains all the Ads contained in the
   - `urlHandler: URLHandler` - Custom urlhandler to be used instead of the default ones [`urlhandlers`](../../src/urlhandlers)
   - `urlhandler: URLHandler` - Fulfills the same purpose as `urlHandler`, which is the preferred parameter to use
   - `resolveAll: Boolean` - Allows you to parse all the ads contained in the VAST or to parse them ad by ad or adPod by adPod (default `true`)
+  - `allowMultipleAds: Boolean` - A Boolean value that identifies whether multiple ads are allowed in the requested VAST response. This will override any value of allowMultipleAds attribute set in the VAST
```

Comment:
Maybe the doc is misleading, it says it will override any value of allowMultipleAds attribute set in the VAST but actually if I pass allowMultipleAds: false to it and the VAST has allowMultipleAds set to true, we will take the one from the VAST

Reply:
Actually, we do want to replace any vast value if allowMultipleAds was provided in the option. Considering the null coalescing operator it will do so 👍